src/components/Form.js

```diff
@@ -83,7 +83,6 @@ const defaultProps = {
     isSubmitButtonVisible: true,
     formState: {
         isLoading: false,
-        errors: null,
```

Comment:
NAB: I assume crash was fixed in onyx repo. Now what's this removal for?

Reply:
Can't find the exact bug ticket at the moment. I know it was one of the bugs that was found during going through the regressions here: https://expensify.slack.com/archives/C01GTK53T8Q/p1690897222550089

Have to leave for vacation in a few minutes so I can't find it right now, sorry 😓

Reply:
Nvm. Hopefully this PR and https://github.com/Expensify/App/pull/18648 would be merged without further change requests from you. Have a good vacation!